### PR TITLE
feat(base): Created the Movie model

### PR DESCRIPTION
**Pull request by forbaokhanh**
Completed the 1-8 on the Build your own API tutorial: setting up the base as well as database design, migrations and models.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -6,5 +6,7 @@ env:
   - NODE_ENV="test"
 script:
   - npm test
-  - npm run enforce
   - npm run lint
+before_script:
+- npm run db:setup:user
+- npm run db:reset
```
Comment (itsachen) [inline .travis.yml]: Nit: Add a newline at the end of the file - its generally [good practice](https://stackoverflow.com/a/729795/659904) to have the last line in a file end in a newline character.

File: db/migrations/.eslintrc [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+  extends: "lob/migrations"
+}
```
Comment (itsachen) [inline db/migrations/.eslintrc]: Nit: Newline here

File: lib/models/movie.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+const Bookshelf = require('../libraries/bookshelf');
+
+module.exports = Bookshelf.Model.extend({
+  tableName: 'movies',
+  serialize: function () {
```
Comment (itsachen) [inline lib/models/movie.js:7]: Nit: Might want to standardize your tab sizes - in general we use 2 spaces for tabs.

File: lib/server.js [hunk 4]
```diff
@@ -12,4 +12,15 @@ const server = new Hapi.Server({
...
+], (err) => {
+  /* istanbul ignore if */
+  if (err) {
+    throw err;
+  }
+});
+
+module.exports = server;
+
+
```
Comment (itsachen) [inline lib/server.js]: Nit: Looks like theres an newline here

File: test/models/movie.test.js [hunk 5]
```diff
@@ -0,0 +1 @@
+'use '
```
Comment (itsachen) [inline test/models/movie.test.js]: Looks like this made its way into the PR before it was done?

**Review by itsachen — CHANGES_REQUESTED** [review]
Could you add `.DS_Store` to your `.gitignore`? Just a couple of nits - could you reassign to me after?

File: .travis.yml [hunk 6]
```diff
@@ -8,3 +8,7 @@ script:
   - npm test
   - npm run enforce
   - npm run lint
+before_script:
+  - npm run db:setup:user
+  - npm run db:reset
+
```
Comment (itsachen) [inline .travis.yml]: Nit: Missing newline

**Review by itsachen — APPROVED** [review]
Just one remaining nit, other than that LGTM!

File: .gitignore [hunk 7]
```diff
@@ -57,3 +57,5 @@ typings/
 # dotenv environment variables file
 .env
 
+.DS_Store
+
```
Comment (sjl2) [inline .gitignore]: nit: extra line can be deleted.

File: config/development.js [hunk 8]
```diff
@@ -1,5 +1,10 @@
 'use strict';
 
 module.exports = {
-  PORT: 3000
+  PORT: 3000,
```
Comment (sjl2) [inline config/development.js]: nit: alphabetize. Your text editor should have something that does this.

File: config/test.js [hunk 9]
```diff
@@ -1,5 +1,10 @@
 'use strict';
 
 module.exports = {
-  PORT: 3000
+  PORT: 3000,
```
Comment (sjl2) [inline config/test.js]: nit: alphabetize

File: package.json [hunk 10]
```diff
@@ -6,6 +6,13 @@
   "scripts": {
     "test": "NODE_ENV=test istanbul cover _mocha -- test --require test/setup.js --recursive --timeout 30000",
     "lint": "eslint .",
+    "db:migrate": "knex migrate:latest --knexfile db/index.js",
```
Comment (sjl2) [inline package.json:9]: nit: alphabetize

File: test/models/movie.test.js [hunk 11]
```diff
@@ -0,0 +1,22 @@
...
+
+    it('includes all of the necessary fields', () => {
+      const movie = Movie.forge().serialize();
+
+      expect(movie).to.have.all.keys([
+        'id',
+        'title',
+        'release_year',
+        'object'
+        ]);
```
Comment (sjl2) [inline test/models/movie.test.js]: nit: I think the `]` is suppose to line up with `expect`, but I'm not 100%. @itsachen ??

**Review by sjl2 — APPROVED** [review]
LGTM with a couple nits!

File: test/models/movie.test.js (hunk 11 above)
Comment (itsachen) [reply test/models/movie.test.js]: 👍 Yeah nice catch, it should line up

**Pull request merged**